src/users/users.service.ts

```diff
@@ -99,7 +105,7 @@ export class UsersService {
         return user;
     }
 
-    async createUserMultiple(employeeMultiple: []): Promise <any>{
+    async createUserMultiple(employeeMultiple: []): Promise <any> {
```

Comment:
Would we be able to change the return type to be a specific Promise type instead of any?

Reply:
Yea we should def change it to a type, but we should take that up with Nghia since he made that method.